parsec/backend/memory/organization.py

```diff
@@ -103,7 +103,16 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
             if vlob_organization_id == id:
                 data_size += blockmeta.size
 
-        users = len(self._user_component._organizations[id].users)
+        users = 0
+        active_users = 0
+        users_per_profile_detail = self._init_users_per_profile_detail()
```

Comment:
`_init_users_per_profile_detail` shouldn't be declared in the parent class: we should try to limit the parent class to api handling and interface definitions.
Here given the helper is rather short and simple (i.e. `detail = {p.value: {"active": 0, "revoked": 0}for p in UserProfile}`), it's just more readable to keep it inlined ;-)